assets/js/mbta.js

```diff
@@ -0,0 +1,532 @@
+const stationConfig = {
+  Orange: [
+    {
+      id: 'OOAK',
+      name: 'Oak Grove',
+      zones: {
+        n: true, s: true, e: 'Busway', w: false, c: false, m: true,
+      },
+    },
+    {
+      id: 'OMAL',
+      name: 'Malden Center',
+      zones: {
+        n: false, s: false, e: false, w: false, c: false, m: 'Entire Station',
```

Comment:
maybe 'Both Platforms'?  or are there real m signs too

Reply:
Not sure. I took the phrasing from the ID in realtime_signs: `malden_center_entire_station`. Does @dmaltzan know the layout of the station?

Reply:
I think there are some on the mezzanine and some in the center of the platform